extensions/sql/transfer-process-store/doc/ddl_postgres.sql

```diff
@@ -14,15 +15,16 @@ COMMENT ON COLUMN edc_lease.leased_at IS 'posix timestamp of lease';
 
 COMMENT ON COLUMN edc_lease.lease_duration IS 'duration of lease in milliseconds';
 
+DROP TABLE IF EXISTS edc_transfer_process;
 CREATE TABLE IF NOT EXISTS edc_transfer_process
 (
     id                       VARCHAR           NOT NULL
         CONSTRAINT transfer_process_pk
             PRIMARY KEY,
-    type                     INTEGER DEFAULT 0 NOT NULL,
+    type                     INTEGER           NOT NULL,
```

Comment:
These two enums should probably be persisted by their String value to avoid issues with changing enum definitions.

Reply:
good point. will update.